comp/core/workloadmeta/params.go

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package workloadmeta
+
+// Params provides the kind of agent we're instantiating workloadmeta for
+type Params struct {
+	AgentType  AgentType
+	InitHelper InitHelper
```

Comment:
If I understand correctly, we currently have a single `InitHelper` and it's used to initialize the tagger. What is the motivation behind that ?
Also, I think `InitHelper` might be confusing as it looks more like a callback.

Reply:
It is a callback and is needed at the moment. This will be removed as componentization progresses.